### DISCUSSION TITLE
any way to do a simple google login screen?

### DISCUSSION THREAD

**Discussion by joaquingrech** [Bdaya-Dev/oidc#93] [Q&A]
any way to do a simple google login screen? the documentation is lacking, I have no idea where to begin and whether it is even possible with this package. It says it supports OAuth2, which as far as I know it is supported... but I see no examples

**Comment by ahmednfwela** [comment]
hi @joaquingrech please check the example [here](https://github.com/Bdaya-Dev/oidc/tree/main/packages/oidc/example)

you get the client id + secret + discovery document + redirect Uri from google, and plug them [here](https://github.com/Bdaya-Dev/oidc/blob/main/packages/oidc/example/lib/app_state.dart)

**Reply by ahmednfwela** [reply]
make sure to follow the getting started page here:

https://bdaya-dev.github.io/oidc/oidc-getting-started/

**Reply by joaquingrech** [reply]
I followed it verbatim and I was not able to get it to work. I also found nothing in the internet using it to guide me.
A cookbook or basic examples would be greatly appreciated. The current example points to nowhere, a Google login would be awesome.

**Comment by ahmednfwela** [comment]
In regards to the documentation, I was thinking of adding a "cook book" page, which contains guides for common Identity providers and how to handle different scenarios.
Let me know if you have anything in mind.
